docs/data/charts/overview/overview.md

```diff
@@ -7,34 +7,33 @@ packageName: '@mui/x-charts'
 
 # MUI X Charts
 
-<p class="description">A fast and extendable library of react chart components for data visualization.</p>
+<p class="description">A collection of React chart components for data visualization.</p>
 
 {{"component": "@mui/docs/ComponentLinkHeader", "design": false}}
 
 ## Overview
 
-The `@mui/x-charts` is an MIT library for rendering charts relying on [D3.js](https://d3js.org/) for data manipulation and SVG for rendering.
-And, like other MUI X components, charts are production-ready components that integrate smoothly into your app.
+MUI X Charts is a library of production-ready components for rendering charts with React.
+It uses [D3.js](https://d3js.org/) for data manipulation and SVGs for rendering.
 
-With a high level of customization, MUI X Charts provides three levels of customization layers: **single components** with great defaults, extensive **configuration props**, and **subcomponents** for flexible composition.
-Additionally, you can also use all the [MUI System](https://mui.com/system/getting-started/) tools, such as the theme override or the `sx` prop.
+The components provide a high level of customization, with beautiful Material Design-based defaults as well as extensive configuration props and flexible composition options.
```

Comment:
Material Design is a constant issue 
```suggestion
The components provide a high level of customization, with beautiful defaults as well as extensive configuration props and flexible composition options.
```

I’m in favor of removing "_Material Design_" here, since it’s one of the biggest discussions we've been facing regarding the look and feel. Users often find it outdated or unattractive, so it feels odd to use it to showcase how beautiful the components are.

Reply:
Interesting point, I'm cool with that